python/tskit/drawing.py

```diff
@@ -166,7 +166,19 @@ def remap(original_map, new_key, none_value):
 
 class SvgTreeSequence:
     """
-    Draw a TreeSequence in SVG.
+    An SVG representation of a tree sequence. The elements in the tree sequence are
+    placed into different `SVG groups <https://www.w3.org/TR/SVG2/struct.html#Groups>`_
```

Comment:
This documentation isn't exposed to the user (it's a private internal class), so there's no point in elaborating here. Need to move this to the ``draw_svg`` function.

Reply:
Yes, thanks. I wasn't sure where to put them, so dumped them there for the time being. I'll hoick them out and put them in the proper place.